### ISSUE TITLE
Integrate SIP SMS MMS messaging with stock Android

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sonuarya/csipsimple#2869]
```
As it stands csipsimple messaging (SMS and MMS) does not interface to the stock 
android messaging system.  Is there any way to either make csipsimple the 
default messaging system for android so that other apps (gallery, contacts, 
etc.) have csipsimple as an option under "share" to send SMS MMS messages?

This would also allow SIP phone users to load 3rd party SMS and MMS apps to 
send SMS and MMS messages.

So basically what is needed is a wrapper for the csipsimple app that translates 
incoming and outgoing SMS and MMS messages into the standard android format.  
Any suggestions?
```

Original issue reported on code.google.com by `jsherk...@gmail.com` on 11 Jan 2015 at 6:24
